extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/EnvVarHolder.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.kubernetes.deployment;
 
+import static org.apache.commons.lang3.StringUtils.EMPTY;
```

Comment:
I would say let's not depend on anything Apache Commons if we don't absolutely have too

Reply:
Deleted.